docker-compose.yml

```
@@ -0,0 +1,10 @@
+services:
+  web:
+    build:
+      context: .
+      args:
+        version: v2024.6.12
+    container_name: harmony-container
```

Personal preference, but the container suffix seems redundant to me. :)
```suggestion
    container_name: harmony
```